acquisition/README.md

```diff
@@ -10,7 +13,7 @@ pip install -r requirements-dev.txt
 
 ```bash
 cd <tomviz_repo>/acquisition
-python -m tomviz
+python -m tomviz.acquisition.cli
```

Comment:
This could be `tomviz-acquisition`

Reply:
I think it is fine for now, and works, we need more work on this but at least it can be started form the guide now.